src/server/routes/api/v1/parking-lot.ts

```diff
@@ -0,0 +1,105 @@
+import httpResponse from '../../../utils/http-response';
+import type {
+  FastifyError,
+  FastifyInstance,
+  FastifyRegisterOptions,
+  FastifyReply,
+  FastifyRequest,
+} from 'fastify';
+import { ParkingLotNotFound } from '../../../error/parking-lot.service';
+
+export default function (
+  fastify: FastifyInstance,
+  _: FastifyRegisterOptions<unknown>,
+  done: (err?: FastifyError) => void,
+): void {
+  fastify.post(
+    '/',
+    async (
+      request: FastifyRequest<{
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      try {
+        const parkingLot = await fastify.services.parkingLot.create(
+          request.body,
```

Comment:
This request body could be anything here, I'm looking forward to have HTTP body validations before handling values to internal layers such as services in this case.

Reply:
This should be resolved when schema is implemented! ☝🏻